src/NuGet.Clients/NuGet.PackageManagement.UI/PackageInstallationInfo.cs

```diff
@@ -49,6 +49,26 @@ private async ValueTask InitializeAsync(CancellationToken cancellationToken)
         private NuGetVersion _versionInstalled;
         private string _versionRequested;
 
+        private int _installedVersionMaxVulnerability = -1;
+        public int InstalledVersionMaxVulnerability
+        {
+            get => _installedVersionMaxVulnerability;
+            set
+            {
+                if (_installedVersionMaxVulnerability != value)
+                {
+                    _installedVersionMaxVulnerability = value;
+                    OnPropertyChanged(nameof(InstalledVersionMaxVulnerability));
+                    OnPropertyChanged(nameof(IsInstalledVersionVulnerable));
+                }
+            }
+        }
+
+        public bool IsInstalledVersionVulnerable
+        {
+            get => InstalledVersionMaxVulnerability > -1;
```

Comment:
Why does have 0 vulnerabilities imply that the installed version is vulnerable?

Reply:
Vulnerability Severity can be from 0 to 3, Low, Moderate, High, Critical

Reply:
In other words, the 0 represents the severity of the vulnerability, not the number of vulnerabilities it has.

Reply:
got it! thanks 😊